tqp/flask.py

```diff
@@ -1,14 +1,44 @@
+from flask import _request_ctx_stack as context_stack
+
 from .topic_queue_poller import TopicQueuePoller
 
 # -----------------------------------------------------------------------------
 
+UNDEFINED = object()
+CTX_PAYLOAD_KEY = 'payload'
+
+# -----------------------------------------------------------------------------
+
+
+@property
```

Comment:
uhh i don't think you can do this?

what?

Reply:
lol yeah. I was writing a class and then refactored this. oops